src/stages/evaluate.ts

```diff
@@ -331,6 +337,7 @@ function getMissingSlotsOnSlotStatus<T, G>(
  */
 function getUnfilledSlots<T, G>(
   getState: () => WolfState,
+  slots: Slot<G>[],
   abilities: Ability<T, G>[],
```

Comment:
Parameter `slots` and `abilities` can be simplified to pass in `flow` which contains both. Simplifies parameter from 2 -> 1